lisa/trace.py

```diff
@@ -1512,22 +1547,68 @@ class FtraceCollector(devlib.FtraceCollector, Loggable, Configurable):
 
     CONF_CLASS = FtraceConf
 
-    def __init__(self, target, events=None, functions=None, buffer_size=10240, autoreport=False, **kwargs):
+    def __init__(self, target, events=None, functions=None, buffer_size=10240, autoreport=False, trace_clock=None, saved_cmdlines_nr=4096, tracer=None, **kwargs):
         events = events or []
         functions = functions or []
-        kwargs.update(dict(
+        trace_clock = trace_clock or 'global'
```

Comment:
Do we have an idea of the overhead/limitations of using the global clock?

Reply:
it might be slower. That said on ARM arch, the system clock is a architecturally a system-wide clock so I expect that using `global` will make no difference compared to `local`. On x86, that might change. Trappy now expects monotonic timestamps anyway, so using `local` clock sounds like a pretty bad idea.